### ISSUE TITLE
Deploy RIALTO Core + Derivative Datastore to AWS Prod Env

### ISSUE THREAD

**Issue by cmh2166** [sul-dlss/rialto#124]
Blocked by #123

**Comment by aaron-collier** [comment]
Closing. Dupe.

**Comment by jcoyne** [comment]
@aaron-collier what is this a duplicate of?  This ticket was blocking #125, but I don't know how to update that ticket now that this is closed.
